.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,24 @@
 <!--
-Thank you for contributing to manim!
-
-Please ensure that your pull request works with the latest
-version of manim from this repository.
+Thank you for your interest in contributing to ManimCommunity!
+
+Before filling out the details below, please ensure:
+- Your local changes are up-to-date with ManimCommunity/manim's latest
+changes and any conflicts are resolved.
```

Comment:
```suggestion
Please ensure that your pull request works with the latest
version of ManimCommunity/manim.
```
@jsonvillanueva: I d like to make this first sentence as short as possible, so the chance that people will read the following keyword section is higher.

Reply:
I'd prefer to keep the bullet-point nature of this section but agree it could be more concise. Maybe this should also be part of the checklist in the [# Acknowledgement](https://github.com/ManimCommunity/manim/pull/1073/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R44) heading?

Reply:
After looking at how[ NumPy does it](https://numpy.org/devdocs/dev/development_workflow.html#writing-the-commit-message), we could also have this in our contributing guidelines on the docs since the `# Acknowledgement` section links to it.

Reply:
I've made it shorter, adopted NumPy acronyms, and added the checklist items. Lmk what you think @kolibril13

Reply:
> I'd prefer to keep the bullet-point nature

I agree, lets do it like you said.